.github/workflows/mac_release.yml

```diff
@@ -54,6 +62,9 @@ jobs:
         echo "Using brew to install mactex and adding it to PATH"
         brew install --cask mactex
         echo "/Library/TeX/texbin" >> $GITHUB_PATH
+        # The MACOSX_DEPLOYMENT_TARGET environment variable sets the default value for the CMAKE_OSX_DEPLOYMENT_TARGET variable.
+        # We use cmake commands to build some subprojects, so setting it globally
+        echo MACOSX_DEPLOYMENT_TARGET=${{ matrix.macos_dev_target }} >> $GITHUB_ENV
```

Comment:
This could potentially affect stuff in third_party the fortran thing, so might as well be consistent.